src/site/markdown/server/manage-webadmin.md

```diff
@@ -1545,6 +1545,45 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
+## Creating address regex
+You can use **webadmin** to create regex mappings.
+
+When a request containing a JSON body with two fields `source` and `regex` is sent, the route will create the regex mapping.
+
+For example: send a request with body
+```
+{
+    "source": "james@domain.tld",
+    "regex": "^[aeiou]"
+}
+```
+the route will add a source `james@domain.tld` with a regex mapping `^[aeiou]` into RecipientRewriteTable.
+This feature uses [Recipients rewrite table](/server/config-recipientrewritetable.html) and requires
```

Comment:
I bet you are missing line breaks between your paragraphs...

Reply:
Space added.